cost-analyzer/values-instance-types.yaml

```diff
@@ -1,5 +1,7 @@
 instanceTypes:
-  enabled: true
```

Comment:
I dont think we need enabled flag.

Reply:
Just checking existence of providers seems fine to me